datafusion/physical-plan/src/joins/sort_merge_join.rs

```diff
@@ -2280,22 +2280,22 @@ fn fetch_right_columns_from_batch_by_idxs(
             .map(|column| take(column, &buffered_indices, None))
             .collect::<Result<Vec<_>, ArrowError>>()
             .map_err(Into::<DataFusionError>::into)?),
-        // If the batch was spilled to disk, less likely
-        (Some(spill_file), None) => {
-            let mut buffered_cols: Vec<ArrayRef> =
-                Vec::with_capacity(buffered_indices.len());
-
-            let file = BufReader::new(File::open(spill_file.path())?);
-            let reader = StreamReader::try_new(file, None)?;
-
-            for batch in reader {
-                batch?.columns().iter().for_each(|column| {
-                    buffered_cols.extend(take(column, &buffered_indices, None))
-                });
-            }
+            // If the batch was spilled to disk, less likely
```

Comment:
I don't understand why this code was changed (seems like the whitespace changes are not needed)